### DISCUSSION TITLE
1.0.38

### DISCUSSION THREAD

**Discussion by sylikc** [sylikc/jpegview#32] [zzArchive]
#### Bugs fixed:
- INI: Removed quotes around ColorCorrection setting.
  Values enclosed in quotes are ignored and hardcoded defaults are used.
  PR #12, thanks [oughtsz](https://github.com/oughtsz)
#### New features:
- Adds transparency support for WEBP format.
  Honors "TransparencyColor" INI setting.
  PR #27, thanks [KrokusPokus](https://github.com/KrokusPokus)
#### Other changes:
- Updated Greek translation, thanks [George](https://sourceforge.net/u/geonetor/)
- Updated Chinese Simplified translation, thanks [dukeng](https://github.com/dunkeng)
- Rename Russian INI template to match localized INI naming convention
- Added LICENSE.txt to explicitly specify the GPLv2 copyright

<hr /><em>This discussion was created from the release <a href='https://github.com/sylikc/jpegview/releases/tag/v1.0.38'>1.0.38</a>.</em>

**Comment by bitsper2nd** [comment]
Is there any chance this gets published on Winget? It is the default package manager on Windows 11.

**Reply by sylikc** [reply]
Great suggestion.  I'll look into it.  Let me read some docs and get back to you on this

**Reply by sylikc** [reply]
@bitsper2nd I've started work on this... but first I need to make an installer for JPEGView... it'll take some time before I get to the Winget part but let's see how it goes

**Reply by bitsper2nd** [reply]
This is nice to read.

**Reply by sylikc** [reply]
Alright @bitsper2nd I think I have some working installer... it's probably going to be a throwback to like an installer you see 20 years ago... not pretty by any stretch of imagination... but it works lol.

I actually need it tested.  I'm testing on Win10 x64, and it seems to work fine... it took me a few weeks to get the shortcuts and associations working.

I can build one and put it in a pre-release to see if it works for a few people.  What do you think?  

I'm going to look into the winget stuff now that I actually have an MSI to work with

... This is the first time I've worked with the MSI building... and it looks pretty crappy lol.  As a graphics designer, you might want to shield your eyes haha

![install](https://user-images.githubusercontent.com/1364494/158012823-bb229482-63d2-454d-b2c5-365e49776385.PNG)
![uninstall](https://user-images.githubusercontent.com/1364494/158012855-98fa7435-5f32-461e-9d19-0a006208f7a7.PNG)

**Reply by sylikc** [reply]
I've made the release for testing.  https://github.com/sylikc/jpegview/releases/tag/v1.0.39.1-wix

You can continue the discussion here or on that release discussion

**Reply by bitsper2nd** [reply]
It looks fine to me on Windows 11. That said, once the package becomes available on winget, we will barely see the installer. 
![image](https://user-images.githubusercontent.com/22452474/158022368-2beec812-bb52-4e68-8199-79afcc68522f.png)

Thanks for the effort you put into this. No problems from my side.

![image](https://user-images.githubusercontent.com/22452474/158022533-83068cef-7542-4ecc-a982-0a54d4bd2113.png)
Programs and Feats

![image](https://user-images.githubusercontent.com/22452474/158022558-cca77616-ca57-4826-872f-5a82aca43852.png)
Start Menu
